src/form-engine.component.tsx

```diff
@@ -289,6 +305,7 @@ const FormEngine: React.FC<FormProps> = ({
                       <EncounterForm
                         formJson={refinedFormJson}
                         patient={patient}
+                        patientPrograms={hasPatientWorkflows ? activePatientEnrollment : null}
```

Comment:
I think we can let the `activePatientenrollment` always be available since it won't interfere with any other code, this is just so we can skip the step above of having to traverse the form if the only reason is to check the `programWorkflow` rendering from fields.